app-backend/app/src/main/resources/application.conf

```diff
@@ -41,6 +41,12 @@ auth0 {
   # Auth0 Bearer with proper permissions for Management API
   bearer = ""
   bearer = ${?AUTH0_MANAGEMENT_BEARER}
+
+  managementClientId = ""
+  managementClientId = ${?AUTH0_MANAGEMENT_CLIENT_ID}
```

Comment:
These settings correspond to the Auth0 non-interactive client, right? How did you authorize it to use the management API? I think we'll have to do that across the three Auth0 accounts for RF.